addon/services/unified-event-handler.js

```diff
@@ -217,18 +218,37 @@ export default Ember.Service.extend(Ember.Evented, {
     return targetHandlers && targetHandlers[eventName] || undefined;
   },
 
+  /**
+   * Creates a function that is used as a handler when throttling events. This makes it possible
+   * for the same throttler function to be used for the same event name
+   * @private
+   * @return {function}
+   */
+  _generateThrottledTriggerFn() {
+    return function({ target, eventName } = {}) {
```

Comment:
should probably return an arrow function since we require valid `this` anyways (then we can tweak the `run.throttle` to avoid re-binding...

Reply:
Spoke offline. Throttle can only be called with a target (target, fn). Going to leave as-is